packages/clerk-js/src/ui/customizables/makeResponsive.tsx

```diff
@@ -33,7 +33,7 @@ export const makeResponsive = <P extends React.JSX.IntrinsicElements['img']>(
 
 const CLERK_IMAGE_URL_BASES = [
   'https://img.clerk.com/',
-  'https://img.clerk.dev/',
+  'https://img.clerk.com/',
```

Comment:
❓ Maybe keep this one for backwards compatibility? If not, then `.com` entry already exists.

Reply:
let us proceed with keeping only the `.com`. Monitor the logs and then drop the `img.clerk.dev`  domain after some time (eg 6 months) if most of the traffic is migrated. wdyt?